fvm/fvm_test.go

```diff
@@ -3652,7 +3652,9 @@ func TestSettingExecutionWeights(t *testing.T) {
 		fvm.WithTransactionFee(fvm.DefaultTransactionFees),
 		fvm.WithExecutionEffortWeights(
 			weightedMeter.ExecutionEffortWeights{
-				common.ComputationKindStatement: 1 << weightedMeter.MeterExecutionInternalPrecisionBytes,
+				common.ComputationKindStatement:          1 << weightedMeter.MeterExecutionInternalPrecisionBytes,
+				common.ComputationKindLoop:               0,
+				common.ComputationKindFunctionInvocation: 0,
```

Comment:
not sure why need these zero, wasn't the case these were counted as 1 before?

Reply:
The test is intended as if these were 0. Which means that after this change I need to explicitly set them to 0 in the state, to override the FWM hardcoded defaults (which are 1)